src/parser/Lexer.h

```diff
@@ -107,6 +107,8 @@ enum PunctuatorKind : uint8_t {
     LogicalNullishEqual,
     SubstitutionEnd,
 
+    HashBang,
```

Comment:
Can you arrange these in alphabetical order? This is an opinion based on my personal preference, so you can ignore it :)

Reply:
I reordered as alphabetical order

Reply:
Thanks!